katas/args/andres/parser.js

```diff
@@ -0,0 +1,47 @@
+import { Argument } from './argument';
+
+export class Parser {
+    constructor (newSchemaString) {
+        this.schemaArray = this.splitArgumentList(newSchemaString); // splits and filters the schema args string
+        this.flagArray = this.takeArgumentList(this.schemaArray); // stores an array of arguments
+
+    }
+    returnFlags () {
+        return this.flagArray;
+    }
+    returnSchema () {
+        return this.schemaArray;
+    }
```

Comment:
It helps to readability to add an empty line between the end and start of functions